pyanaconda/payload/rpmostreepayload.py

```diff
@@ -53,6 +54,14 @@ def __init__(self, *args, **kwargs):
         self._internal_mounts = []
         self._locale_map = None
 
+        # flatpak_payload will be None if it can't be used in this environment (don't have repo)
+        self._flatpak_payload = None
+
+        if FlatpakPayload.is_available():
```

Comment:
There could be a new type of the `RPMOSTreePayload` with the flatpack support always enabled, so you don't have to check whether `self._flatpak_payload` is None all the time.

Reply:
Great solution. Working on that!